### PR TITLE
PWN-6186 - Add Firebase, Appsflyer Tracker, for log events

### PR DESCRIPTION
**Pull request by Davran331122**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-6186

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -67,12 +69,14 @@ android {
             signingConfig signingConfigs.debug
             minifyEnabled true
             shrinkResources true
-            versionNameSuffix ".${getVersionBuild()}-feature"
+            versionNameSuffix ".${getVersionBuild() }-feature"
```
Comment (gslevinkov) [inline app/build.gradle:72]: whitespace?

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -26,6 +26,7 @@
         android:dataExtractionRules="@xml/data_backup_rules"
         android:fullBackupContent="@xml/backup_rules"
         android:icon="@mipmap/ic_launcher"
+        android:largeHeap="true"
```
Comment (gslevinkov) [inline app/src/main/AndroidManifest.xml:29]: left from debug?

File: app/src/main/java/org/p2p/wallet/common/analytics/trackers/AppsFlyerTracker.kt [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import android.content.Context
+import com.appsflyer.AppsFlyerLib
+import com.appsflyer.attribution.AppsFlyerRequestListener
+import timber.log.Timber
+
+class AppsFlyerTracker(
+    private val context: Context
+) : AnalyticsTracker {
+
+    private val appsflyer = AppsFlyerLib.getInstance()
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/common/analytics/trackers/AppsFlyerTracker.kt:12]: appsFlyer

File: app/src/main/java/org/p2p/wallet/common/analytics/trackers/AppsFlyerTracker.kt [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    private val requestListener = object : AppsFlyerRequestListener {
+        override fun onSuccess() {
+            Timber.tag("AppsFlyerTracker").d("Event successfully logged")
+        }
+
+        override fun onError(p0: Int, message: String) {
+            Timber.tag("AppsFlyerTracker").d("Error while try to log event: $message")
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/common/analytics/trackers/AppsFlyerTracker.kt:19]: not .e?

File: app/src/main/java/org/p2p/wallet/common/analytics/trackers/FirebaseTracker.kt [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+import android.content.Context
+import com.google.firebase.analytics.FirebaseAnalytics
+import com.google.firebase.analytics.ktx.logEvent
+import org.koin.core.parameter.parametersOf
+
+class FirebaseTracker(
+    private val context: Context
+) : AnalyticsTracker {
+
+    private val firebaseAnalytics = FirebaseAnalytics.getInstance(context)
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/common/analytics/trackers/FirebaseTracker.kt:12]: please inject FirebaseAnalytics instead of Context

**Review by kabramovp2p — APPROVED** [review]

**Review by gslevinkov — APPROVED** [review]

**Pull request merged**